src/qibotn/backends/cpu.py

```diff
@@ -0,0 +1,144 @@
+from qibo.backends.numpy import NumpyBackend
+from qibo.config import raise_error
+from qibo.states import CircuitResult
+
+
+class QuTensorNet(NumpyBackend):
```

Comment:
```suggestion
class QuimbBackend(NumpyBackend):
```
?

Below you're calling it like that.
https://github.com/qiboteam/qibotn/blob/890b9d1c93fc981efc2d2fee998b3f8ca2ef071f/src/qibotn/backends/cpu.py#L115